main/zoe/guide/contracts/autoswap.md

```diff
@@ -13,33 +13,27 @@ An implementation of [UniSwap](https://uniswap.org/).
 
 ## Initialization
 
-Create an instance of an autoswap invite:
+Create an instance of the Autoswap code, and receive an invite that
+when redeemed, will let you add liquidity to Autoswap.
```

Comment:
just replace "redeemed" with "used"? or "receive an invite that will let you..."